package.json

```diff
@@ -44,26 +44,26 @@
     "prepare": "husky install"
   },
   "imperative": {
-    "configurationModule": "lib/imperative.js"
+    "configurationModule": "lib/pluginDef.js"
```

Comment:
oh, this is a good precedent for V3 🥳 
Thank you! 💯 
Maybe then we stop calling everything the `Imperative` file 😋